src/bot.js

```diff
@@ -15,14 +15,14 @@ class Bot {
 
 		// TODO: get list of rooms from |queryresponse|rooms to determine if
 		// they're public or not.
-		// this.roomsData = Set();
+		// this.roomsData = new Set();
```

Comment:
I only figured it was an ES5 set because there was no `import {Set} from 'immutable';` - then again, since the line was commented out, it doesn't need to be there. My bad